src/Sylius/Bundle/GridBundle/DependencyInjection/Compiler/RegisterDriversPass.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\GridBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class RegisterDriversPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $registry = $container->getDefinition('sylius.registry.grid_driver');
```

Comment:
Compiler passes should not fail if container is empty. `findDefinition` should be used instead of `getDefinition` as only the first one supports aliases.

Anyway, integration test for this pass would be nice, there are some examples in `@ThemeBundle/Test/Translation/DependencyInjection/Compiler`.